### DISCUSSION TITLE
download project error

### DISCUSSION THREAD

**Discussion by mrzhang638993** [Netflix/conductor#3176] [Q&A]
#13 [conductor:server builder 4/4] RUN ./gradlew build -x test --stacktrace
#13 10.72
#13 10.72 Exception in thread "main" java.io.IOException: Downloading from https://services.gradle.org/distributions/gradle-7.5.1-bin.zip failed: timeout
#13 10.72       at org.gradle.wrapper.Download.downloadInternal(Download.java:122)
#13 10.72       at org.gradle.wrapper.Download.download(Download.java:80)
#13 10.72       at org.gradle.wrapper.Install$1.call(Install.java:83)
#13 10.72       at org.gradle.wrapper.Install$1.call(Install.java:63)
#13 10.72       at org.gradle.wrapper.ExclusiveFileAccessManager.access(ExclusiveFileAccessManager.java:69)
#13 10.72       at org.gradle.wrapper.Install.createDist(Install.java:63)
#13 10.72       at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:107)
#13 10.72       at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:63)
#13 10.73 Caused by: java.net.SocketTimeoutException: connect timed out
#13 10.73       at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
#13 10.73       at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:412)
#13 10.73       at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:255)
#13 10.73       at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:237)
#13 10.73       at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
#13 10.73       at java.base/java.net.Socket.connect(Socket.java:609)
#13 10.73       at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:305)
#13 10.73       at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:177)
#13 10.73       at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:508)
#13 10.73       at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:603)
#13 10.73       at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:266)
#13 10.73       at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:373)
#13 10.73       at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:207)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1081)
#13 10.73       at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:193)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1592)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1520)
#13 10.73       at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:250)
#13 10.73       at org.gradle.wrapper.Download.downloadInternal(Download.java:100)
#13 10.73       ... 7 more
#13 ERROR: executor failed running [/bin/sh -c ./gradlew build -x test --stacktrace]: exit code: 1

#14 [conductor:ui internal] load metadata for docker.io/library/node:14-alpine
#14 CANCELED
------
 > [conductor:server builder 4/4] RUN ./gradlew build -x test --stacktrace:
#13 10.73       at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:373)
#13 10.73       at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:207)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1081)
#13 10.73       at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:193)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1592)
#13 10.73       at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1520)
#13 10.73       at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:250)
#13 10.73       at org.gradle.wrapper.Download.downloadInternal(Download.java:100)
#13 10.73       ... 7 more
------
failed to solve: executor failed running [/bin/sh -c ./gradlew build -x test --stacktrace]: exit code: 1

**Comment by mrzhang638993** [comment]
information: https://conductor.netflix.com/gettingstarted/docker.html

**Comment by jegadjame** [comment]
I am also facing the same problem.. Is the issue fixed

**Comment by jxu-nflx** [comment]
Hello @mrzhang638993 can you check your network configures? is it behind any proxy? You also might want to check your docker network settings to make sure it has proper setup to connect to the internet.

**Comment by jegadjame** [comment]
Hi,
Mine is under proxy and able to connect to internet. I was able to download elastic search image
